CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## stable-2.6.1
+
+This release improves proxy stability by fixing a bug where the proxy could stop
+receiving service discovery updates, resulting in 503 errors
```

Comment:
smallest nit ever, but we typically end the intro paragraph in a punctuation mark (omitting punctuation marks at the end of each bullet point)